wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -493,9 +525,11 @@ instance Arbitrary NodeInfo where
 --
 
 type family Update (original :: *) :: * where
-  Update Wallet  = WalletUpdate
-  Update Account = AccountUpdate
+  Update Wallet        = WalletUpdate
+  Update Account       = AccountUpdate
+  Update WalletAddress = () -- read-only
```

Comment:
I know we had this discussion already but cannot remember :trollface: Can we omit `Update WalletAddress` at all from the type family? If we can, I'm tempted not to bother, as it's more confusing writing something that maps to `()` than not write anything.

Reply:
Yep, it is needed for `readOnlyFieldsFromJSON` to work, we decided that defining this type instance would be easier than writing another `allFieldsFromJSON`